pages/transcripts/react-native-core-team.md

```diff
@@ -0,0 +1,682 @@
+---
+title: React Native Core Team
+date: May 9 2019
+---
+
+## Follow the team
+
+[@Eli_White](https://twitter.com/Eli_White) (EliWhite)
+[@fkgozali](https://twitter.com/fkgozali) (fkgozali)
```

Comment:
can we put fkgozali's name in the parentheses here? `(Kevin Gozali)`

Reply:
```suggestion
[@fkgozali](https://twitter.com/fkgozali) (fkgozali) (Kevin Gozali)
```

Some thing like this?

Reply:
Added change.

✅